src/auro-icon.js

```diff
@@ -95,14 +100,19 @@ class AuroIcon extends AuroElement {
    * @returns {dom} DOM ready HTML to be appended
    */
   async fetchIcon(category, name) {
-    const icon = await fetch(`https://unpkg.com/@alaskaairux/icons@3.4.0/dist/icons/${category}/${name}.svg`);
+    const icon = await fetch(`https://unpkg.com/@alaskaairux/icons@4.0.1/dist/${category}/${name}.svg`);
```

Comment:
This is a breaking change as it removes "icons" as the default from the URL. Due to this, consumers will have to include "icons" as part of their category name.

For example
```
<auro-icon category="alert" name="error" accent></auro-icon>
```
would need to be changed to
```
<auro-icon category="icons/alert" name="error" accent></auro-icon>
```

An alternative to this which wouldn't be a breaking change is adding "icons" dynamically based on the category. Suggestions are welcome.